### DISCUSSION TITLE
Initial brainstorm of centres/tribes/communities of interest in the Second Renaissance ecosystem

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#975] [Research]
Would love to get feedback on our first quick sketch of "Tribes"/"Centres" of the Second Renaissance ecosystem ... 
(Done with @mattosborn) 

- What would you add? (+ why)
- What would you take out? (+ why)
- What surprises you to see?
- & any other reflections most welcome!

![Screenshot 2023-12-21 at 16 46 59](https://github.com/life-itself/community/assets/97066487/4cbe5895-2ae4-4031-bead-99ce3e030897)

**Comment by matthewmccarthy11** [comment]
What has stuck out to me as a particular tribe in the cohere+ project not mentioned here are the 'global consciousness' or 'global unity' kind of groups and people. They're connected to 'spiritual' but not necessarily the same. This includes orgs like charter for compassion, earth charter, earthwise, converge, weavinglab,  maybe  even the hague center. They have a focus on the 'development of a global consciousness' and include peace organizations, networking organizations, educational and activist ones. Sometimes one might even be confused about 'what they actually do' which is why some orgs, like whole world view, were not included in the cohere criteria; they often have 'rainbows'  and globes in their designs; they often use words like 'gaia' and 'consciousness' and I would say people like Ervin Laszlo and Deepak Chopra are leading the community. I would say a significant amount of organizations exist that have this 'particular style', or atleast a flavor of it, and the example that I would give is whole world view (we called it a 'maybe') https://www.wholeworld-view.org/partners . On this page you can also find related orgs.

**Comment by matthewmccarthy11** [comment]
Also I think it might be an interesting idea to instead of going from 'category' to include different orgs, starting from 'orgs' as 'centers'- like for example, I might include these two as 'centers':

Reos partners: https://reospartners.com/

Maybe, 'consulting firms which appear to be genuinely concerned with systems transformation?

The consilience project: https://consilienceproject.org/

Orgs like 'the alternative' from the uk

What exactly is the center? I don't know, but it seems to be representative of a certain kind of organization

**Comment by martinprogrammer** [comment]
There's a growing number of people who, through listening to the likes of Tim Ferris and his guests (for example) come across all those notions included in the sketch.  I think that media of this sort acts as a community because they curate the content to create their own flavour of an eclectic mix of these progressive teachings, leanings and ideas.

But these people themselves, might not be part of any organisation, either physically or philosophically - yet they are an important part of this eco-system because they successfully appeal to the lot outside of the reach of the spiritual movement.

There are wealthy businessmen, successful professionals and sports people who have, to an extent,  all adopted some positive aspects of the above and can exert influence.  

What do you think?

**Comment by asimong** [comment]
Before I answer your actual questions, @catherinet1 I'd like to clarify a couple of things. First, what does the spatial orientation represent? Is it an attempt to represent conceptual closeness, and if so, is that just for you (two) or is it based on some wider survey? Second, related, are there meant to be, or could there be, dimensions on the chart? Many such charts have dimensions in two dimensions. Maybe there are more as well... but as you haven't labelled the edges or axes it looks like you aren't trying to do that, right?

**Comment by james-baker** [comment]
Looking around for [Liminal Web](https://www.joelightfoot.org/post/the-liminal-web-mapping-an-emergent-subculture-of-sensemakers-meta-theorists-systems-poets), Game B, Metacrisis (Schmachtenberger, Perspectiva, Vervaeke), Economic-systems-but-not-Game-B-ers (Nate Hagens, Raworth, Doomer Optimists)
